adafruit_ra8875/ra8875.py

```diff
@@ -341,9 +347,10 @@ def touch_init(
         :param bool enable: Enable the Touch Functionality as well
         """
         if tpin is not None:
-            tpin.direction = Direction.INPUT
+            tpin.switch_to_input()
         self._tpin = tpin
-        self._write_reg(reg.INTC2, reg.INTC2_TP)
+        # Clear the Interrupt Flag
+        # self._write_reg(reg.INTC2, reg.INTC2_TP)
```

Comment:
Is this suppose to be commented out? If so, might help to add more info to code comment as to why.

Reply:
Good point. It wasn't really necessary. I'll just remove it.